api/client/container/stop.go

```diff
@@ -40,16 +41,35 @@ func NewStopCommand(dockerCli *client.DockerCli) *cobra.Command {
 
 func runStop(dockerCli *client.DockerCli, opts *stopOptions) error {
 	ctx := context.Background()
+	timeout := time.Duration(opts.time) * time.Second
 
+	var waitAll sync.WaitGroup
+	var printLock sync.Mutex
+	var errLock sync.Mutex
 	var errs []string
+
+	var maxParallel int32 = 1000
```

Comment:
Sorry, I don't quite get your suggestion. The stop has a timeout, so what "timeout" are we talking about?